test/cpp/jit/test_misc.h

```diff
@@ -748,6 +749,56 @@ graph(%a):
   AT_ASSERT(testPassValue);
 }
 
+static void checkShape(Node* n, std::vector<int64_t> expected) {
+  auto tp = n->output()->type();
+  auto ptp = tp->expect<ProfiledTensorType>();
+  ASSERT_EQ(ptp->sizes().concrete_sizes().value(), expected);
+}
+
+void testProfiler() {
```

Comment:
Do we really need an entire LSTM to test a profiler? Could we get a smaller test? It's a bit hard to tell what the profiler does from this test.